particles/Tutorial/Kotlin/Demo/src/total/TTTHumanPlayer.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.tutorials.tictactoe
+
+import arcs.sdk.Handle
+
+class TTTHumanPlayer : AbstractTTTHumanPlayer() {
```

Comment:
Diff shows this file being introduced with this patch, which doesn't seem right. Any ideas why?

Reply:
No idea how it snuck in... I got rid of it.